### PR TITLE
Phase 3: OS settings and browsers

### PR DESCRIPTION
**Pull request by joeyparrish**
This adds packages for Linux, macOS, and Windows to configure recommended settings and install all major browsers.

### CODE REVIEW

File: shaka-lab-recommended-settings/windows/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+choco uninstall -y shaka-lab-recommended-settings
+```
+
+## Specific Settings and Dependencies
+
+Dependencies:
+ - vim (text editor)
+
+Settings:
+ - Install and start SSH server
```
Comment (alekzz) [inline shaka-lab-recommended-settings/windows/README.md]: This might need some hints like which SSH server to use and where to download it.

**Review by alekzz — DISMISSED** [review]

File: shaka-lab-recommended-settings/windows/README.md (hunk 1 above)
Comment (joeyparrish) [reply shaka-lab-recommended-settings/windows/README.md]: No, this is not a step the user must take.  It is documentation of steps that are done for you by the package.  In this case, it installs and starts the SSH server provided by Microsoft.  I'll make some updates to the doc to clarify.

**Comment by joeyparrish** [comment]
@alekzz, please let me know if you think this is an improvement: https://github.com/shaka-project/shaka-lab/pull/17/commits/7eac8d739ffbe31d19b9ee2514293076f0e48e64

File: shaka-lab-recommended-settings/windows/README.md (hunk 1 above)
Comment (alekzz) [reply shaka-lab-recommended-settings/windows/README.md]: Got it. Thank you for changing the titles, now it's clear.

**Review by alekzz — APPROVED** [review]

**Pull request merged**